templates/payments/workshop_list.html

```diff
@@ -2,6 +2,14 @@
 
 {% block title %}QIIME - Workshops{% endblock %}
 {% block header %}Workshops{% endblock %}
+{% block extrahead %}
+<!--  Legal requirement for EU viewable websites https://www.cookielaw.org/the-cookie-law/ -->
```

Comment:
I don't know if we actually need this: the EU cookie law exemption criteria allows for first-party cookies that are used as "[user-input](http://ec.europa.eu/ipg/basics/legal/cookies/index_en.htm#section_2)" cookies.